config/locales/en.yml

```diff
@@ -205,6 +205,7 @@ en:
     search:
       heading: 'Browse and search for works'
       placeholder: 'Search for titles, keywords, resource types, creators, etc.'
+      label: 'Search Scholarsphere'
```

Comment:
Shouldn't this be inter-capped a la #467

Reply:
@rschenk No, because it's a screen-reader only thing. BUT, if it ever wasn't I supposed we might want to...

Reply:
@rschenk meh. I changed it anyway. I don't expect it'll affect the screen reader at all.